cinema_service/urls.py

```diff
@@ -2,10 +2,17 @@
 from django.conf.urls.static import static
 from django.contrib import admin
 from django.urls import path, include
+from drf_spectacular.views import SpectacularSwaggerView, SpectacularAPIView
 
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
     path("api/user/", include("user.urls", namespace="user")),
```

Comment:
Ensure that the `user.urls` module is correctly defined and includes the necessary URL patterns for the user app, with the namespace 'user'.